httpfuzzerprocessor/showDifferences.js

```diff
@@ -0,0 +1,76 @@
+//A Fuzzer HTTP Processor script that compares the original Response with the fuzzed Response 
+//and add the result to the state column!
+ 
+var DiffTool = Java.type("org.zaproxy.zap.extension.diff.diff_match_patch");
+var key = "fuzz.httpfuzzer.messageprocessor.tagcreator.tags.text";
+var showResultInTable = true;
+
+
+function processMessage(utils, message) {
+	return message;
+}
+
+// Called after receiving the fuzzed message from the server
+function processResult(utils, fuzzResult){	
+	var original = responseAsString(utils.getOriginalMessage());
+	var fuzzed = responseAsString(fuzzResult.getHttpMessage());
+	var diffList = createDiff(original, fuzzed);	
+	var aggregatedDiff = aggregateDiff(diffList);	
+	displayToStateColumn(fuzzResult, aggregatedDiff);
+	return showResultInTable;
+}
+
+function responseAsString(httpMessage){
+	var responseHeader = httpMessage.getResponseHeader().toString();
+	var responseBody = httpMessage.getResponseBody().toString();
+	return responseHeader + "\r\n" + responseBody;
+}
+
+function createDiff(original, fuzzed){
+	var diffTool = new DiffTool();
+	return diffTool.diff_main(original, fuzzed);
+}
+
+function displayToStateColumn(fuzzResult, aggregatedDiff){
+	removeAllStates(fuzzResult);
```

Comment:
Latest version of the fuzzer add-on does show multiple states, they are no longer overwritten (per zaproxy/zaproxy#3166, it just shows one icon though).